### PR TITLE
Updating Enterprise component versions for v3.5.0 release.

### PR DESCRIPTION
**Pull request by stevegaossou**
## Description
- Pin enterprise versions to v3.5.0
- Update repo path

## For PR author

- [ ] Tests for change.
- [ ] If changing pkg/apis/, run `make gen-files`
- [ ] If changing versions, run `make gen-versions`

## For PR reviewers

A note for code reviewers - all pull requests must have the following:

- [ ] Milestone set according to targeted release.
- [ ] Appropriate labels:
  - `kind/bug` if this is a bugfix.
  - `kind/enhancement` if this is a a new feature.
  - `enterprise` if this PR applies to Calico Enterprise only.

### CODE REVIEW

File: pkg/components/images.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2020 Tigera, Inc. All rights reserved.
+// Copyright (c) 2021 Tigera, Inc. All rights reserved.
```
Comment (bcreane) [inline pkg/components/images.go:1]: 2020-2021?

File: config/enterprise_versions.yml [hunk 2]
```diff
@@ -46,42 +46,40 @@ components:
...
+    version: v3.5.0
   compliance-server:
     image: tigera/compliance-server
-    version: release-calient-v3.5
+    version: v3.5.0
   compliance-benchmarker:
     image: tigera/compliance-benchmarker
-    version: release-calient-v3.5
-  # Below components are all maintained by 3rd parties and thus specify a 'registry' to indicate
-  # to the docs & manifest tooling not to default to the tigera registry.
```
Comment (stevegaossou) [inline config/enterprise_versions.yml:75]: This comment block is old and no longer relevant.

**Review by doublek — APPROVED** [review]
✅

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Comment by marvin-tigera** [comment]
I'm sorry but I failed delete the branch after merging the pull request.

**Pull request merged**